creator-node/src/components/healthCheck/syncHealthCheckComponentService.js

```diff
@@ -1,30 +1,33 @@
-const { priorityMap } = require('../../snapbackSM')
-
-const getJobInfo = job => ({
-  type: job.name,
-  priority: priorityMap[job.opts.priority],
+const getJobInfo = (job) => ({
   id: job.id,
-  wallet: job.data.syncRequestParameters.data.wallet[0],
-  secondary: job.data.syncRequestParameters.baseURL
+  secondary: job.data.syncRequestParameters.baseURL,
+  syncType: job.data.syncRequestParameters.data.syncType,
+  wallet: job.data.syncRequestParameters.data.wallet[0]
 })
 
-const makeResponse = (pendingJobs, activeJobs) => ({
-  pending: pendingJobs.map(getJobInfo),
-  active: activeJobs.map(getJobInfo),
-  pendingCount: pendingJobs.length
+const makeResponse = (manualWaitingJobs, manualActiveJobs, recurringWaitingJobs, recurringActiveJobs) => ({
+  manualWaiting: manualWaitingJobs.map(getJobInfo),
+  manualActive: manualActiveJobs.map(getJobInfo),
+  recurringWaiting: recurringWaitingJobs.map(getJobInfo),
+  recurringActive: recurringActiveJobs.map(getJobInfo),
+  manualWaitingCount: manualWaitingJobs.length,
+  recurringWaitingCount: recurringWaitingJobs.length
 })
 
 /**
  * Returns information about sync queue.
  * Response: {
- *  pending: Array<{ type, priority, id }>,
- *  active: Array<{ type, priority, id }>,
- *  pendingCount: number
+ *  manualWaiting: Array<{ type, id }>,
+ *  manualActive: Array<{ type, id }>,
+ *  recurringWaiting: Array<{ type, id }>,
```

Comment:
is `type` redundant here? Like shouldn't everything in manual be 'manual' and same for recurring?

Reply:
still worth keeping this exposed imo just clarifying

Reply:
nah you right, it is redundant, fixed